frontend/src/services/transactions/token.ts

```diff
@@ -9,9 +9,9 @@ import {
 } from '@/services/transactions/utils';
 import type { IEthereumProvider } from '@/services/web3-provider';
 import type { EthereumAddress, Token, TransactionHash } from '@/types/data';
-import type { MintableToken } from '@/types/ethers-contracts/goerli';
 import { TokenAmount } from '@/types/token-amount';
 import { UInt256 } from '@/types/uint-256';
+import type { MintableToken } from '#contracts/MintableToken.ts';
```

Comment:
> because it is not part of the "core" beamer contracts
Thats a very good point. It also will never change. Lets move it to the assets folder instead.

Reply:
The only thing that might be confusing is that we end up with multiple MintableToken type definitions inside the `ethers-contracts` folder.

Reply:
I would rather keep it as it is for now and just change the import path to point to the goerli type.

Reply:
Ok sure. This wouldn't be related to this PR anyway

Reply:
Why would there be multiple definitions? Do we also have multiple of the FeeSub types?